.gitignore

```diff
@@ -37,5 +37,6 @@ env-config.sh
 *.db
 
 /calypso-strings.php
```

Comment:
You're right, I left it in to avoid complaints about left-over `calypso-strings.php` but the CircleCI builds are generated fresh anyway.